From ef24126a056361f1ed9797ead748996799c77ef7 Mon Sep 17 00:00:00 2001
From: hongao <hongao@uniontech.com>
Date: Tue, 7 May 2024 15:19:44 +0800
Subject: [PATCH 1/3] feat:intel integrated graphics before 10th  uses intel
 driver instead of modesetting,

install 10-intel.conf in /usr/share/X11/xorg.conf.d/

Signed-off-by: hongao <hongao@uniontech.com>

--- xserver-xorg-video-intel-2.99.917+git20210115.orig/src/i915_pciids.h
+++ xserver-xorg-video-intel-2.99.917+git20210115/src/i915_pciids.h
@@ -533,12 +533,7 @@
 	INTEL_CFL_U_GT3_IDS(info), \
 	INTEL_WHL_U_GT1_IDS(info), \
 	INTEL_WHL_U_GT2_IDS(info), \
-	INTEL_WHL_U_GT3_IDS(info), \
-	INTEL_AML_CFL_GT2_IDS(info), \
-	INTEL_CML_GT1_IDS(info), \
-	INTEL_CML_GT2_IDS(info), \
-	INTEL_CML_U_GT1_IDS(info), \
-	INTEL_CML_U_GT2_IDS(info)
+	INTEL_WHL_U_GT3_IDS(info)
 
 /* CNL */
 #define INTEL_CNL_PORT_F_IDS(info) \
--- xserver-xorg-video-intel-2.99.917+git20210115.orig/src/intel_module.c
+++ xserver-xorg-video-intel-2.99.917+git20210115/src/intel_module.c
@@ -476,12 +476,6 @@ static const struct pci_id_match intel_d
 
 	INTEL_CNL_IDS(&intel_cannonlake_info),
 
-	INTEL_ICL_11_IDS(&intel_icelake_info),
-	INTEL_EHL_IDS(&intel_elkhartlake_info),
-
-	INTEL_TGL_12_IDS(&intel_tigerlake_info),
-
-	INTEL_VGA_DEVICE(PCI_MATCH_ANY, &intel_generic_info),
 #endif
 
 	{},
